fastembed/text/e5_onnx_embedding.py

```diff
@@ -0,0 +1,67 @@
+from typing import Type, List, Dict, Any
+
+import numpy as np
+
+from fastembed.text.onnx_embedding import OnnxTextEmbedding, OnnxTextEmbeddingWorker, EmbeddingWorker
+
+supported_multilingual_e5_models = [
+    {
+        "model": "intfloat/multilingual-e5-large",
+        "dim": 1024,
+        "description": "Multilingual model, e5-large. Recommend using this model for non-English languages",
+        "size_in_GB": 2.24,
+        "sources": {
+            "url": "https://storage.googleapis.com/qdrant-fastembed/fast-multilingual-e5-large.tar.gz",
+            "hf": "qdrant/multilingual-e5-large-onnx",
+        },
+    },
+    {
+        "model": "xenova/multilingual-e5-large-quantized",
+        "dim": 1024,
+        "description": "Multilingual model. Recommended for non-English languages",
+        "size_in_GB": 2.24,
+        "sources": {
+            "hf": "xenova/multilingual-e5-large",
+        }
+    },
+    {
+        "model": "xenova/paraphrase-multilingual-mpnet-base-v2",
```

Comment:
```suggestion
        "model": "sentence-transformers/paraphrase-multilingual-mpnet-base-v2",
```

This is the model name. I missed this in the PR from Anush. Apologies for the confusion.